README.md

```diff
@@ -46,11 +46,12 @@ Federico Menozzi <federicogmenozzi@gmail.com>
 Desktop client for Shadertoy
 
 USAGE:
-    shadertoy [OPTIONS] [shader] [SUBCOMMAND]
+    shadertoy [FLAGS] [OPTIONS] [shader] [SUBCOMMAND]
 
 FLAGS:
-    -h, --help       Prints help information
-    -V, --version    Prints version information
+        --force_srgb_off    Forces srgb to be off. (Use this you have color blending issues)
```

Comment:
For the help string here and in `src/cli.yml`, what about something like "Forces SRGB to be off (replicates shadertoy.com color blending)" ?

Reply:
Yep, sound better, updated !

Reply:
Thanks!